Makefile

```diff
@@ -34,7 +36,7 @@ help:
 	@echo "docker-testall:      run all tests (in container)"
 	@echo "docker-run:          run the development webserver (in container)"
 
-html: for-documentable
+html: for-documentable bigpage
 	documentable start -a -v --highlight
```

Comment:
This should probably be better left removed. bigpage can simply be an independent target.

Reply:
I agree that `bigpage` should be a completely independent target. OTOH I think it would be nice if the bigpage version of docs.raku.org is kept updated (and in sync with the other pages) and I guessed (possibly incorrectly) that whatever automation re-generates it uses the `html` Makefile target. Ideally the automation should build both, independent targets (possibly via a `docs.rakudo.org` target, which would also build the `epub` target besides `html` and `bigpage`) - WDYT?

So I removed the dependency in 7779e9a14. Do you know what would be the best way to accomplish the goal of keeping the bigpage version updated?

Reply:
Well, so far updating is done by hand (by YT). I'll do that by hand, from time to time, when there's a sufficient amount of change and it's in a stable state. Also, I don't use make, but do that directly. So not automation here.
If you see the roadmap, web generation should be totally independent. When that's done, it'll probably be automated along with the rest, with a huge amount of tests; not for the time being, I'm afraid.